README.md

```diff
@@ -9,3 +9,6 @@
 [![GitHub license](https://img.shields.io/github/license/swissfintechinnovations/ca-mortgage)](https://github.com/swissfintechinnovations/ca-mortgage/blob/main/LICENSE)
 
 This is the official SFTI repo for the mortgage API. Documentations may be found in the [Wiki](https://github.com/swissfintechinnovations/ca-mortgage/wiki).
+
+We refer to the SwaggerEditor for a graphical representation and source-file based documentation of the latest API state:
```

Comment:
May I suggest the following wording instead: "An easy-to-read representation of the Mortgage API is accessible via the following link (please note that if you click on the following link, you will leave GitHub unless you use your browser's "open link in new tab/window" feature):"?